### DISCUSSION TITLE
Modification of a single part of the presentation field

### DISCUSSION THREAD

**Discussion by PTKaroveggia** [globaleaks/globaleaks-whistleblowing-software#3707] [Q&A]
Is there a way to change a single word in the presentation field?
By edit I mean line break or write in bold

**Comment by evilaliv3** [comment]
Thank you for the question. Would you please add a clear example to clarify what you would like to do?

**Comment by lakshaa** [comment]
You can modify it using CSS, which worked for me. I have placed the text in the presentation field and then worked on the CSS to display it like I wanted.
